infra/charts/feast-feature-server/templates/deployment.yaml

```diff
@@ -36,6 +36,8 @@ spec:
           env:
             - name: FEATURE_STORE_YAML_BASE64
               value: {{ .Values.feature_store_yaml_base64 }}
+            - name: INTRA_COMMUNICATION_BASE64
+              value: {{ "intra-server-communication" | b64enc }}
```

Comment:
where this values come from  ->  `{{ "intra-server-communication" | b64enc }}`

Reply:
Its a hard coded value at the moment

Reply:
@tokoko what do you think introducing new environment variable  `INTRA_COMMUNICATION_BASE64` or we can add this property inside feature_store.yaml under auth section. The draw back I am seeing client can configure the same property to hack this.

Reply:
The other approach is configure/create the full access roles for the servers but that again will cause the performance issue and server keep generating the token or fetching the roles .

Reply:
> Its a hard coded value at the moment

I see will will converted into base64 automatically when will be deployed ?

Reply:
That was @dmartinol suggestion

Reply:
> The draw back I am seeing client can configure the same property to hack this.

I don't really see the difference, but env variable sounds better.